client/my-sites/checkout/src/hooks/use-tos-foldable-card.tsx

```diff
@@ -1,7 +1,15 @@
-import { hasCheckoutVersion } from '@automattic/wpcom-checkout';
+import { useExperiment } from 'calypso/lib/explat';
 
 export function useToSFoldableCard(): boolean {
-	if ( hasCheckoutVersion( 'tos-foldable-card' ) ) {
+	const [ isLoadingExperimentAssignment, experimentAssignment ] = useExperiment(
+		'wp_web_checkout_tos_foldable_card_v1'
```

Comment:
We're going to need to also use the `isEligible` option to `useExperiment` to disable this for Jetpack and Akismet checkout